test/cli/test_volargparse.py

```diff
@@ -0,0 +1,297 @@
+import pytest
+
+from volatility3 import framework, plugins
+from volatility3.cli.volargparse import find_parser
+
+
+@pytest.mark.parametrize(
```

Comment:
This takes up a data and I worry whether we need to keep it up to do (or whether we will).  I think it's related to the plugin name shortening (which as I've said, creates more complexity than it's worth), so I think this whole file can be dropped (although I really appreciate you adding the test cases!)